charactersheet/charactersheet/services/common/account/messaging/node_service.js

```diff
@@ -170,11 +170,7 @@ function _NodeService(config) {
     self._handlePresence = function(receivedPresence) {
         /**eslint no-console:0 */
         try {
-            var xItem = $(receivedPresence).find('x item')[0];
-            if (!xItem) {
-                return true;
-            }
-            var jid = $(xItem).attr('jid');
+            var jid = $(receivedPresence).find('x item').attr('jid');
```

Comment:
It was a passing enhancement. (I guess I forgot to commit it to my Chat Fixes PR).

It just simplifies the logic a bit. It does exactly the same thing.